### PR TITLE
[BEAM-4864] Make run_validatescontainer.sh not quite as hard coded.

### PR DESCRIPTION
**Pull request by holdenk**
Allow users to specify project and bucket for testing their containers in Python.

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ X ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | --- | --- | --- | ---

### CODE REVIEW

**Comment by holdenk** [comment]
Jenkins retest this please.

**Comment by holdenk** [comment]
cc @alanmyrvold the original author (I think) of the script

**Comment by holdenk** [comment]
Ping?

File: sdks/python/container/run_validatescontainer.sh [hunk 1]
```diff
@@ -25,14 +25,16 @@
 #
 # Execute from the root of the repository: sdks/python/container/run_validatescontainer.sh
 
+echo "This script must be executed in the root of beam project. Please set LOCAL_PATH, GCS_LCOATION, and PROJECT as desired."
```
Comment (alanmyrvold) [inline sdks/python/container/run_validatescontainer.sh]: GCS_LOCATION

Comment (holdenk) [reply sdks/python/container/run_validatescontainer.sh]: Thanks!

**Comment by holdenk** [comment]
cc @robertwb has an approval from the original author, if you have some cycles would appreciate review or merge if appropriate.

**Comment by apilloud** [comment]
LGTM. Rerunning tests because its been so long.

**Comment by apilloud** [comment]
run python precommit

**Pull request merged**